doc/user_guide/user_guide.md

```diff
@@ -99,6 +100,39 @@ Rules:
 
 * The Maven file must contain all necessary plugins and settings. Please check a [template](templates/maven_central_release_pom_file_template.md).
 
+### Rules for Release on Exasol Community Portal
+
+* The project must contain a `community_portal_post_template.json` file in the root directory. This json file has a few required objects:
+
+1. Array of strings `tags`: one or more tag of the project to be added to the community post;
+1. String `project name`: this is how the project name will be displayed in the post header;
+1. String `project description`: a short description of the project. Please, use markdown in the description;
+
+An example of the file:
+
+```json
+{
+  "tags": [
+    "Release Droid Testing",
```

Comment:
I used json because I already have a json library in the project and I work with other json files. And switching to yml means adding an additional library which we usually trying to avoid. I don't think it really matters that some files are yml, and this one is json, I think all developers know how to write a json

Reply:
I think user experience should be more important than reducing libraries. 
Knowing how to write it is the one thing. YAML allows you to write and review the config quickly without counting the breakets.
YAML also supports multi-line strings (https://www.baeldung.com/yaml-multi-line) which makes nested markdown more readble.